src/components/sbb-alert-group/sbb-alert-group.tsx

```diff
@@ -5,15 +5,15 @@ import { customElement, property, state } from 'lit/decorators.js';
 import { EventEmitter, ConnectedAbortController } from '../../global/eventing';
 import { SbbAlert } from '../sbb-alert';
 import { setAttribute } from '../../global/dom';
-import Style from './sbb-alert-group.scss?lit&inline';
+import style from './sbb-alert-group.scss?lit&inline';
 
 /**
- * @slot unnamed - content slot, should be filled with `sbb-alert` items.
+ * @slot - content slot, should be filled with `sbb-alert` items.
```

Comment:
I'm not sure whether I like this notation without unnamed but still with "-". Is this a must?

Reply:
As discussed via Teams, this is the recommendation. (Which is still vague, but exists.)